README.md

```diff
@@ -103,6 +103,24 @@ For sample implementations, see the [.NET Core Example](https://github.com/sendg
 
 - Please see the [.nuspec file](https://github.com/sendgrid/sendgrid-csharp/tree/master/nuspec).
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```

Comment:
I think this should be `COPY` as it is possible the user is in a windows terminal. 

https://www.wikihow.com/Copy-Files-in-Command-Prompt